snapshots/overlay/overlay_test.go

```diff
@@ -2,13 +2,10 @@
 
 /*
    Copyright The containerd Authors.
-
```

Comment:
Use this pls: https://github.com/containerd/project/blob/master/script/validate/template/go.txt